tests/utils/managed_process.py

```diff
@@ -59,6 +60,7 @@ def __enter__(self):
 
             self._terminate_existing()
             self._start_process()
+            time.sleep(self.delayed_start)
```

Comment:
curious do we need delayed start or could increase timeout?

Reply:
Please see the issue here: https://github.com/ai-dynamo/dynamo/issues/1419